SDKGenerator.njsproj

```diff
@@ -197,6 +197,9 @@
     <Content Include="targets\csharp\source\source\WsaReflectionExtensions.cs" />
     <Content Include="targets\csharp\templates\API.cs.ejs" />
     <Content Include="targets\csharp\templates\APIInstance.cs.ejs" />
+    <Content Include="targets\csharp\source\source\Extensions\Models\PlayFabCloudScriptModelsExtensions.cs.ejs">
+      <SubType>Code</SubType>
```

Comment:
Can we remove the SubType here so this file matches the others? If so, please remove.

Reply:
Done